test/e2e/quick_start_test.go

```diff
@@ -99,3 +99,17 @@ var _ = Describe("When following the Cluster API quick-start with Ignition", fun
 		}
 	})
 })
+
+// NOTE: This test requires a dualstack management cluster (can be configured via IP_FAMILY=dual).
+var _ = Describe("When following the Cluster API quick-start with dualstack [dualstack] [PR-Informing]", func() {
+	QuickStartSpec(ctx, func() QuickStartSpecInput {
+		return QuickStartSpecInput{
+			E2EConfig:             e2eConfig,
+			ClusterctlConfigPath:  clusterctlConfigPath,
+			BootstrapClusterProxy: bootstrapClusterProxy,
```

Comment:
How do we confirm that dual stack works in the cluster we create here?

Is it basically that ApplyClusterTemplateAndWait ensure that the Nodes exist?

Reply:
Current test is really a smoke-test  - similar to the existing IPv6 test. We just create a cluster with a mix of IPs, but don't ensure that dual stack connectivity etc. is working.